src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBox/TextBox_Disabled_State.xaml

```diff
@@ -18,11 +18,13 @@
 				<StackPanel>
 					<ToggleSwitch IsOn="{Binding [IsEnabled], Mode=TwoWay}"
                                   OnContent="TextBox is Enabled"
```

Comment:
Align properties with IsOn (Ctrl + K & Ctrl + D, in order to format the document) ;)